documentation/docs/reference/software/interfaces/mqtt.md

```diff
@@ -59,171 +109,390 @@ Control of the focus stage. The message is a JSON object, speed is optional:
+
+**Authorized values for `setting` camera parameters action:**
 
-Status is one of `Started`, `Ready`, `Done`, `Interrupted`, `Error`, `Dead`.
+| Parameter             | Type   | Accepted Values      | Description                                    |
+|-----------------------|--------|----------------------|------------------------------------------------|
+| `action`              | string | "settings"           | Specifies the action to update camera settings.|
+| `iso`                 | int    |  100 to 800          | ISO sensitivity value.                         |
+| `shutter_speed`       | int    |  125 to 1000  (in μs)           | Shutter speed value.                           |
+| `WB:red`  | object |      0.0 to 32.0    | White balance gain values for red.    |
+| `WB:blue`  | object |       0.0 to 64.0     | White balance gain values for blue.    |
+| `white_balance`       | string | "auto", "off"            | White balance mode.                            |
+| `image_gain`          | object |                      | Image gain values for analog and digital.      |
 
-- Publish only
+**Status/Error Messages for `settings` action:**
 
-#### `status/imager`
+| Status/Error                                      | Cause                                                                                                     |
+|---------------------------------------------------|-----------------------------------------------------------------------------------------------------------|
+| `"Camera settings updated"`         | The camera settings have been successfully updated.                                                       |
+| `"Camera settings error"`           | The settings command is missing required parameters.                                                      |
+| `"Error: Resolution not valid"`     | The provided resolution parameter is invalid.                                                             |
+| `"Error: Iso number not valid"`     | The provided ISO parameter is invalid.                                                                    |
+| `"Error: Shutter speed not valid"`  | The provided shutter speed parameter is invalid.                                                          |
+| `"Error: White balance gain not valid"` | The provided white balance gain parameters are invalid.                                                   |
+| `"Error: White balance mode %s not valid"` | The provided white balance mode parameter is invalid.                                                      |
+| `"Error: Image gain not valid"`     | The provided image gain parameters are invalid.                                                           |
+| `"Busy"`                            | The camera is currently busy and cannot update settings.                                                  |
 
-State of the imager. It's a JSON object with:
+- **JSON message to stop the imager**:
```

Comment:
For each command on this page, I would like to see one-sentence or two-sentence description of what that command does, under the section header for that command. If you're not sure what to write, you can use the descriptions I had written in https://docs.google.com/document/d/1BxAlUBMRXEhJkk6OMDVYN7z7l7mky7YTwA_C2Uykrm4/edit#heading=h.fmuu43juukhx as a starting point. For example, for this `stop` command my description was:

> Stops any in-progress stop-flow sample dataset acquisition routine.